### PR TITLE
add USER_ID_FIELD environment variable to be used as key to map X-Has…

### PR DESCRIPTION
**Pull request by mshanak**
…ura-User-Id

### CODE REVIEW

File: src/token.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
+const config = require('./config');
+const userid_field_name = config.get('UserIdField');
```
Comment (httpsOmkar) [inline src/token.js]: Prefer camel case `userIdFieldName`

File: src/token.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 exports.tokenParser = (content, clientId, debugMode) => {
     const accessToken = content.access_token;
 
-    const userId = accessToken.content.sub;
+    const userId = accessToken.content[userid_field_name+''];
```
Comment (httpsOmkar) [inline src/token.js]: `+' '` the space can return some undefined

**Comment by mshanak** [comment]
update the field name

**Comment by mshanak** [comment]
removed the  +' '

**Comment by httpsOmkar** [comment]
Thanks for MR

**Pull request merged**